docs/guides/sdk/authentication.md

```diff
@@ -53,7 +53,7 @@ import { createDirectus, realtime } from '@directus/sdk';
 
 const client = createDirectus('http://directus.example.com').with(realtime());
 
-client.send(JSON.stringify({ type: 'auth', email: email, password: password }));
+client.sendMessage(JSON.stringify({ type: 'auth', email: email, password: password }));
```

Comment:
Should probably be 

```suggestion
client.sendMessage({ type: 'auth', email: email, password: password });
```

@br41nslug

Reply:
Both approaches are supported by the function but for the docs i do agree lets leave away the manual stringify.